pkg/maps/lrs/lrs.go

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package lrs
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/byteorder"
+	"github.com/cilium/cilium/pkg/ebpf"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/cilium/cilium/pkg/service"
+)
+
+const LoadReportingMapName = "cilium_lb_lrs"
+
+var LoadReportingMap *bpf.Map
+
+func Init() {
```

Comment:
If we're adding this from scratch, it would be preferable to build this using cell.Cell from the get-go to avoid having to use a global variable and doing future refactors.

There's a couple examples of this available in the `pkg/maps`directory, for example [nodemap](https://github.com/cilium/cilium/blob/5b1dce19047f35effa2dae3a270ad1e139f10479/pkg/maps/nodemap/cell.go#L21-L64).